cast/src/lib.rs

```diff
@@ -717,6 +717,11 @@ pub enum InterfacePath {
     Etherscan { address: Address, chain: Chain, api_key: String },
 }
 
+pub enum BindPath {
+    Local { path: String, name: Option<String> },
+    Etherscan { address: Address, chain: Chain, api_key: String },
+}
+
```

Comment:
needs docs

Reply:
Created this just for separation of concern, but it is kind of doing the same thing as **InterfacePath**. Since both are actually holding location of ABI, be it local or etherscan, how about refactoring it all together to **AbiPath**?

Reply:
yes lets unify